src/rollup/data/tracking/get-data.js

```diff
@@ -0,0 +1,7 @@
+/** @module pathfora/data/tracking/get-data */
+
+import { pathforaDataObject } from '../../globals/config';
+
+export default function getData () {
```

Comment:
@markhayden yeah, I think I only kept the name here cause it's a public method. But I doubt anyone is using it besides our tests so I don't feel like there's any harm in changing it.